autoload/utils/exec/job.vim

```diff
@@ -152,7 +140,9 @@ function! utils#exec#job#run(cmd, err_fmt) abort
                     \ 'cmd': a:cmd
                     \ }
     else
-        let l:job = job_start(a:cmd, {
+        let l:cmd = has('win32') ? a:cmd : ['/bin/sh', '-c',  a:cmd]
```

Comment:
@mark2185 Why do you run command in new shell?
Maybe if we use new shell we can add `noglob` without exception for `job` executor.

Reply:
Well it needs to be a `List` for `Unix`, but it cannot be just `[ a:cmd ]` because the documentation says:
```
		{command} can be a List, where the first item is the executable
		and further items are the arguments.
```
But yes, I could use `&shell` and add `s:add_noglob` for that. That should work.